l2-contracts/contracts/dev-contracts/TimestampAsserter.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.20;
+
+interface ITimestampAsserter {
+    function assertTimestampInRange(uint256 start, uint256 end) external view;
+}
+
+contract TimestampAsserter is ITimestampAsserter {
+    function assertTimestampInRange(uint256 start, uint256 end) public view {
+        require(
+            start <= block.timestamp && block.timestamp <= end,
+            "Block timestamp is outside of the specified range"
```

Comment:
we try to use [custom errors](https://soliditylang.org/blog/2021/04/21/custom-errors/) in code

Reply:
Fixed